src/components/Common/ManagerWrapper/ManagerWrapper.tsx

```diff
@@ -1,12 +1,10 @@
-import ClearIcon from '@mui/icons-material/Clear';
 import './ManagerWrapper.scss';
 
-const ManagerWrapper = ({ children, closeManagerPage }: { children: JSX.Element; closeManagerPage: () => void; }) => {
+const ManagerWrapper = ({ children }: { children: JSX.Element;}) => {
 
     return (
         children ?
             <div className='manager-wrapper'>
-                <ClearIcon className='close-icon' onClick={closeManagerPage} />
```

Comment:
why dont we need the X?

Reply:
just didn't look good in my opinion, you can now close the catalogs when clicking again on the same buttons